.env

```
@@ -29,7 +29,7 @@ MANAGER_KEY=${PWD}/ocrd/manager/.ssh/id_rsa
 MANAGER_DATA=${PWD}/kitodo/data/metadata
 
 # Application Kitodo.Production
-APP_IMAGE=markusweigelt/kitodo-production-ocrd:latest
+APP_IMAGE=ghcr.io/slub/kitodo-production_ocrd/kitodo-production:latest
```

Mixed dashes and underscores?

I understand we are completely free here?

```suggestion
APP_IMAGE=ghcr.io/slub/kitodo_production_ocrd/kitodo-production:latest
```

---

Note: I did not use `kitodo_production:latest`, because I cannot make suggestions for the corresponding tag line in publish.yml. But being consistent is preferable, so you may want to use underscores there as well.

---

I changed that cause "Kitodo" and "Production" belongs together but i can change this back to only underscore.